Y/Yosys/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Yosys"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/YosysHQ/yosys.git", "2e421feb0ea526468493ab008f3c72beb12c2bc6")
+]
+
+dependencies = [Dependency("boost_jll"),
+                Dependency("Tcl_jll"),
+                Dependency("Readline_jll"),
+                Dependency("Libffi_jll")]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd yosys
+make -j3; make install PREFIX=$prefix
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; experimental=true)
+platforms = expand_cxxstring_abis(platforms)
+
+# The products that we will ensure are always built
+products = Product[
+	ExecutableProduct("yosys", :yosys)
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version=v"10")
```

Comment:
GCC 10 is a bit too much, especially for a C++ program

Reply:
Should I drop that line all together?